index.html

```diff
@@ -48,8 +50,238 @@ <h1 class="main-name">Erik Stoupignan</h1>
         <div class="prueba">
             <img src="/assets/image_geometry_2.png" alt="Decorative photo" class="background-img2">
         </div>
-
     </main>
+
+    <!-- Proyects section -->
+    <h2>Projects</h2>
+    
+    <div class="project">
+
+            <!-- Section  #1-->
+            <div class="container">
+                <img class="box1" src="/assets/Rectangle21.png" alt="Decoration photo">
+                <div class="box box2">
+                    <p class="subtitle">Project names goes here</p>
+                </div>
+                <!-- Button's Code -->
+                <div class="box align box3">
+                    <p class="code-button">HTML/CSS</p>
+                </div>
+                <div class="box align box4">
+                    <p class="code-button">Ruby on Rails</p>
+                </div>
+                <div class="box align box5">
+                    <p class="code-button">JavaScript</p>
+                </div>
+                <!-- See this project button -->
+                <div class="box align-button box6">
+                    <a href="#">
+                        <div class="buttom-arrow">
+                            <p>See this project
+                            <img class="arrow" src="/assets/211621_c_right_arrow_icon.png" alt="Right arrow">
+                            </p>
+                        </div>
+                    </a>
+                </div>
+            </div>
+
+            <!-- Section  #2-->
+            <div class="container">
+                <img class="box1" src="/assets/Rectangle21-2.png" alt="Decoration photo">
+                <div class="box box2">
+                    <p class="subtitle">Project names goes here</p>
+                </div>
+                <!-- Button's Code -->
+                <div class="box align box3">
```

Comment:
- [ ] Kindly change the list of the project technologies to a list `ul` and for each of them, a `li`.

Reply:
Ok, I fixed. Thank you for your advice.